src/sonic-config-engine/sonic-cfggen

```diff
@@ -229,7 +229,7 @@ def main():
         paths  = ['/', '/usr/share/sonic/templates', os.path.dirname(template_file)]
         loader = jinja2.FileSystemLoader(paths)
 
-        env = jinja2.Environment(loader=loader, trim_blocks=True)
+        env = jinja2.Environment(loader=loader, trim_blocks=True, extensions=['jinja2.ext.do'])
```

Comment:
why do we need this jinja2.ext.do?

Reply:
https://github.com/Azure/sonic-buildimage/pull/2838/files#diff-4c9b3f6e87e24e7256cd7abf5e0da806R97

We need a temp variable to store the interfaces that belong to the specified vnet. In this situation, the "do"  action will be needed.